src/main/kotlin/org/misarch/tax/graphql/model/TaxRateVersion.kt

```diff
@@ -0,0 +1,33 @@
+package org.misarch.tax.graphql.model
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+import com.expediagroup.graphql.generator.federation.directives.FieldSet
+import com.expediagroup.graphql.generator.federation.directives.KeyDirective
+import graphql.schema.DataFetchingEnvironment
+import org.misarch.tax.graphql.dataloader.TaxRateDataLoader
+import java.time.OffsetDateTime
+import java.util.*
+import java.util.concurrent.CompletableFuture
+
+@GraphQLDescription("A TaxRateVersion.")
+@KeyDirective(fields = FieldSet("id"))
+class TaxRateVersion(
+    id: UUID,
+    @property:GraphQLDescription("The rate of the TaxRateVersion")
+    val rate: Double,
+    @property:GraphQLDescription("Version of the TaxRateVersion")
+    val version: Int,
+    @property:GraphQLDescription("Time stamp when the TaxRateVersion was created")
+    val createdAt: OffsetDateTime,
+    private val taxRateId: UUID
+) : Node(id) {
+
+    @GraphQLDescription("The associated TaxRate.")
+    fun currentVersion(
```

Comment:
Is that the correct description?
Doesn't sound like what this method is supposed to do…

Reply:
fixed with 822bc48b77cd7baadb1ed3080d6cb95e481f47bd